src/components/one-pager.svelte

```diff
@@ -1,145 +1,188 @@
 <script>
-    import { onMount } from 'svelte'
-    import { GeneralShowInfo } from "../constants";
-    import ArtistsCSV from '../utils/data/artists.csv'
-
-    
-    const random = (min, max) => {
-        return Math.random() * (max - min) + min;
-    }
-    onMount(async () => {
-        const moveCircles = () => {
-            Array.from(document.querySelectorAll(".bubble-overlay")).forEach(svgElement => {
-                const circles = Array.from(svgElement.querySelectorAll(".image-circle"));
-                const cx = random(0, 100);
-                const cy = random(0, 100);
-                // console.log({ cx, cy })
-                circles.forEach(circleEl => {
-                    circleEl.setAttribute("cx", cx);
-                    circleEl.setAttribute("cy", cy)
-                })
-            })
-        }
-        setTimeout(() => {
-            moveCircles();
-            setInterval(moveCircles, 10000)
-        }, 500)
-    })
-
-    function shuffleArray(array) {
-        for (var i = array.length - 1; i > 0; i--) {
-            var j = Math.floor(Math.random() * (i + 1));
-            var temp = array[i];
-            array[i] = array[j];
-            array[j] = temp;
+  import { onMount } from 'svelte'
+  import { GeneralShowInfo } from '../constants'
+  import ArtistsCSV from '../utils/data/artists.csv'
+  import { stores } from '@sapper/app'
+  import UAParser from 'ua-parser-js'
+
+  var parser = new UAParser()
+  const { session } = stores()
+  parser.setUA($session['user-agent'])
+  let isMobile = parser.getResult().device['type'] === 'mobile'
```

Comment:
Is it necessary to find out whether it's mobile? Can't we use media queries and define for size?